src/centriamTable/CentriamTable.js

```diff
@@ -47,11 +49,22 @@ export default class CentriamTable extends React.Component {
         self.state = {
             data: props.data,
             rowHeight: props.rowHeight,
-            rowClick: props.rowClick  || noop, // noop function
             sortInfo: props.sortInfo || new SortInfo(),
-
+            selectedRow: props.selectedRow
         };
 
+        this.rowClick = rowSelection ? function(data){
+            this.setState(Object.assign({}, this.state, {selectedRow : data}));
```

Comment:
I thought setState took care of merging these for you

Reply:
Yes/No. It's supposed to, but I've had problems in the past when merging large objects together. This is just an extremely explicit way of doing it and results in fewer bugs

Reply:
:+1: